cdp_scrapers/instances/portland.py

```diff
@@ -116,6 +117,15 @@ def get_person(self, name: str) -> Optional[Person]:
         #       Should add to known_persons so we don't query this same person again.
         return None
 
+    def separate_name_from_title(self, title_and_name: str):
+        # title_and_name:
+        #   The title (Mayor of Commissioner) and name of a Portland City Commission
+        #   member.
+        #   e.g., Mayor Ted Wheeler, Commissioner Carmen Rubio
+
+        name_index = title_and_name.index(" ")
+        return title_and_name[name_index + 1 :]
+
     def get_matter(self, event_page: BeautifulSoup) -> Optional[Matter]:
```

Comment:
The signature for this method should be changed, now that it is called with an element of a list returned by `BeautifulSoup.find()`. i.e. `event_page: Tag`. See the [`bs4` doc](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#tag) for reference.

Reply:
Oh, the variable name also. `event_page` is no longer correct. Something more relevant.

Reply:
I understand why `event_page` is not good, and I changed it to `minute_section`.

How do you think the signature should be?